### ISSUE TITLE
Cordova coho "tools release" a bit confusing

### ISSUE THREAD

**Issue by brodybits** [apache/cordova-discuss#98]
While trying to help someone else I find the concept of "tools release" as discussed in https://github.com/apache/cordova-coho/blob/master/docs/tools-release-process.md to be a bit confusing.

I would be a bit happier to see packages like cordova-common, cordova-js, and cordova-lib covered separately.

**Comment by janpio** [comment]
(Sidenote: The platform release process is the most up to date and structured, as I invested quite some time to update that one when I did the windows release. Similar work would probably improve the other release processes as well)

**Comment by dpogue** [comment]
The documentation is written with the assumption that a tools release includes all of the tools (which, to be fair, is how it's been done historically).

**Comment by brodybits** [comment]
@dpogue can you give me a pointer to the documentation that you refer to?

**Comment by janpio** [comment]
I assume he refers to the file you linked to - this is the coho documentation.

**Comment by dpogue** [comment]
Yeah, I was referring to https://github.com/apache/cordova-coho/blob/master/docs/tools-release-process.md
